vlib/v/checker/tests/comptime_if_optional_string_test.vv

```diff
@@ -0,0 +1,17 @@
+module main
+
+struct Foo{
+	a string 
+	b ?string
+}
+
+fn test_main(){
+	$for field in Foo.fields {
+		dump(field.name)
+		$if field.typ is ?string {
+			dump(1)
+		} $else $if field.typ is string {
+			dump(2)
+		}
+	}
+}
```

Comment:
Looks like the fmt test needs to test .vv files, too... missing trailing EOL in this file.

Reply:
> Looks like the fmt test needs to test .vv files, too... missing trailing EOL in this file.

Yeah. Good catch. Thanks. 👍🏻